### PR TITLE
[Dark Mode] Update: Search

### PR DESCRIPTION
**Pull request by vquach2404**
This PR updates the Search category to support dark mode.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Search/Reverse geocode/ReverseGeocode.storyboard [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
-                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                             </view>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Tap on the map for addresses" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="9fp-qf-Go1">
-                                <rect key="frame" x="0.0" y="20" width="375" height="44"/>
-                                <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.59866391320000001" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="44"/>
+                                <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Search/Reverse geocode/ReverseGeocode.storyboard:25]: Please use the status label background color for it. Also, remove the height constraint.

File: arcgis-ios-sdk-samples/Search/Find place/FindPlace.storyboard [hunk 2]
```diff
@@ -1,61 +1,59 @@
...
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="59s-fv-YVF" customClass="AGSMapView">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
-                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                             </view>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="M8g-vM-vWe">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
-                                <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.70001492830000001" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
+                                <color key="backgroundColor" white="0.33333333333333331" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Search/Find place/FindPlace.storyboard:25]: overlay

The non-changing Dark gray here look a bit strange. What about using another system gray or the status label bgc?

File: arcgis-ios-sdk-samples/Search/Offline geocode/OfflineGeocode.storyboard [hunk 3]
```diff
@@ -13,22 +14,21 @@
...
-                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                             </view>
-                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="tVG-73-umr">
-                                <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="tVG-73-umr">
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                                 <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.0" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Search/Offline geocode/OfflineGeocode.storyboard:25]: Please set the button's bgc to default.

File: arcgis-ios-sdk-samples/Search/Find address/FindAddress.storyboard [hunk 4]
```diff
@@ -13,22 +14,21 @@
...
-                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                             </view>
-                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="uzi-MY-M2j">
-                                <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="uzi-MY-M2j">
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                                 <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.0" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Search/Find address/FindAddress.storyboard:25]: Reset the bgc to default.

**Review by yo1995 — COMMENTED** [review]
A few comments below.

**Review by yo1995 — DISMISSED** [review]
👍 LGTM. Ready for others to review.
Please capture a new screenshot for `Reverse geocode` if you like. (status label change)

**Review by saratchandrakarumuri — APPROVED** [review]
Changes looks good

**Review by yo1995 — APPROVED** [review]

**Review by philium — APPROVED** [review]

**Pull request merged**